inc/Engine/Admin/RocketInsights/Rest.php

```diff
@@ -620,8 +647,16 @@ private function handle_sync_submission( string $url, bool $is_mobile, array $ad
 			return false;
 		}
 
-		// Update to pending status immediately with job ID.
-		$this->query->make_status_pending( $url, $sync_response['uuid'], '', $is_mobile );
+		Logger::error(
+			'Rocket Insights: Synchronous Submission successful, Now scheduling single job to run in 30 seconds.',
+			[
+				'url' => $url,
+			]
+		);
+
+		// Update to in-progress status immediately with job_id.
+		$this->query->make_status_inprogress( $url, $is_mobile, [ 'job_id' => $sync_response['uuid'] ] );
```

Comment:
why not using the manager class instead of query?
```suggestion
		$this->manager->make_status_inprogress( $url, $is_mobile, 'rocket_insights', [ 'job_id' => $sync_response['uuid'] ] );
```

I like the idea of dealing with the manager as a direct interface to the database layer.